### PR TITLE
Update Table component styles

### PR DESCRIPTION
**Pull request by michal-maksajda**
Issue:  #282 

---

##### PR Checklist
- [x] Give this PR a meaningful title
- [x] Add a link to the related issue at the top of this description (above)
- [x] Connect this PR with the related issue via ZenHub with the button below this text box (or at the bottom of the page after the PR is created)
- [x] Ensure your branch is up to date with the target branch and resolve any conflicts
- [x] Answer the below questions to describe your PR for reviewers
- [x] Request at least two reviewers using the "Reviewers" section on the right, usually including at least one reviewer from your team
- [x] Notify the requested reviewers in the #code-review Slack channel once the PR is ready for review

---

## Why are these changes needed?

According to the design changes and migrating all components to Bootstrap, the Table component styles are now consistent with Bootstrap.

## What changed?

Zrzut ekranu 2021-11-18 o 08 19 16

## How are these changes tested?

- All automatic tests are passing
- Changed might be tested manually by viewing the current version of the deployed version of [storybook](https://fhir-react-lib-test-storybook.s3.amazonaws.com/branch/table-restyle/index.html?path=/story/explanationofbenefit--eob-carin-blue-button-extended-diagnosis)

### CODE REVIEW

**Review by sflejszman — APPROVED** [review]
✅

File: src/components/resources/ExplanationOfBenefit/ExplanationOfBenefit.test.js [hunk 1]
```diff
@@ -195,26 +209,32 @@ describe('should render ExplanationOfBenefit component properly', () => {
...
+      'Encounter for symptom (185345009)',
+      'Acute bronchitis (disorder) (10509002)',
+      'Measurement of respiratory function (procedure) (23426006)',
+    ];
+    const replaceWhitespaces = text => text.replace(/\s+/g, ' ');
+    explanationService.forEach((x, i) => {
+      expect(replaceWhitespaces(x)).toEqual(
+        replaceWhitespaces(expectedArray[i]),
+      );
+    });
```
Comment (beck1uphealth) [inline src/components/resources/ExplanationOfBenefit/ExplanationOfBenefit.test.js:237]: All of the changes here are outside the "it" test blocks, so it seems like they don't run or do anything when I run `npm run test`. Am I missing something? Also, I think this is unrelated but when I run the tests locally, I see a bunch failing that are all because of dates being off by one. Is that because I am in a different timezone?

Comment (michal-maksajda) [reply src/components/resources/ExplanationOfBenefit/ExplanationOfBenefit.test.js:237]: Hi, @beck1uphealth I've checked all of the test suites once more. The test case, which you've mentioned here is running properly on my side. I was testing it by running `yarn test` in the project root directory. 

When I mocked the test case with the incorrect dataset, it correctly reacted to an error.

**Review by beck1uphealth — APPROVED** [review]
